contributors/devel/release.md

```diff
@@ -0,0 +1,246 @@
+# Targeting Features, Issues and PRs to Release Milestones
+
+This document is focused on Kubernetes developers and contributors
```

Comment:
More specifically I think we're just talking about code that is intended to land in kubernetes/kubernetes?  How does this process handle repos like kubernetes/kubeadm?

Reply:
This is an interesting question and currently one without an answer in my opinion.  I don't see it being answered or dictated by this document.  If the document in the meantime gives a description of k/k practice which is embraced by other repos, it would be a positive change toward common experience across the project.

If folks REALLY want the document constrained I could change "Kubernetes developers and contributors" to "kubernetes/kubernetes repo developers and contributors".  But I'll be sad to do it ;)

Reply:
Fair, let's leave it as is and see what happens